### ISSUE TITLE
About Keras bugs[solved]

### ISSUE THREAD

**Issue by poult-lab** [pseeth/soundnet_keras#7]
At first, thanks for wonderful work from author, I appreciate.
I got a lot of bugs when I import keras follow authors. These bugs take me 2 hours. 
We aren't supposed to install Keras separately when you use tf2. Use the Keras bundled with the Tensorflow you have installed.
Finally, I solved it.
We can change soundnet.py to this:
 

import numpy as np
import librosa
from tensorflow.keras.layers import BatchNormalization, Activation, Conv1D, MaxPooling1D, ZeroPadding1D, InputLayer
from tensorflow.keras.models import Sequential
